dataland-keycloak/realms/master-realm.json

```diff
@@ -2385,7 +2385,7 @@
     "parRequestUriLifespan": "60",
     "clientSessionMaxLifespan": "0",
     "frontendUrl": "http://dataland-admin:6789/keycloak",
-    "acr.loa.map": "[]"
+    "acr.loa.map": "{}"
```

Comment:
Does that mean we do not have a list here but a map?

Reply:
I am not quite sure.
I have changes this because there was an error log "ACR LOA MAP" in the keycloak container.
Then I googled this log and people wrote in forums, that this realm setting needs to be in "{}" instead of "[]" from keycloak version 19 on.
That's the only reason I changed it.   The log disappeared after this change.
But I cannot tell you why the keycloak developers changed this or how this influences keycloak services.